homeassistant/components/rpi_power/binary_sensor.py

```diff
@@ -34,7 +34,7 @@ async def async_setup_entry(
 class RaspberryChargerBinarySensor(BinarySensorEntity):
     """Binary sensor representing the rpi power status."""
 
-    _attr_device_class = DEVICE_CLASS_PROBLEM
+    _attr_device_class = BinarySensorDeviceClass.PROBLEM
     _attr_icon = "mdi:raspberry-pi"
```

Comment:
Not directly related to this PR, but do we allow both `device_class` and `icon` to be set?

Reply:
cc @MartinHjelmare

Reply:
I think we can if it's agree that the default icon does not accurately represent the nature of the entity, right?

Reply:
Yes, what Robert said. :+1: